base/src/main/java/org/apache/cassandra/auth/LDAPAuthenticator.java

```diff
@@ -136,6 +137,15 @@ public AuthenticatedUser authenticate(String username, String password) throws A
     {
         try
         {
+
+            if (!Boolean.parseBoolean(properties.getProperty(ALLOW_BLANK_PASSWORD_PROP, "true")))
+            {
+                if (password == null || password.trim().isEmpty())
+                {
+                throw new AuthenticationException("blank password is not supported");
```

Comment:
indentation, also, Cassandra's codestyle is such that if there is just one statement, you do not need braces (` { } `) around it. Just indent it under `if`.

Reply:
changed it, thanks